weave/trace_server/migrations/005_add_cost.up.sql

```diff
@@ -0,0 +1,39 @@
+CREATE TABLE llm_token_prices (
+    /*
+    `pricing_level`: The level at which the token pricing applies. It can be 'default' for global pricing,
+    'project' for project-specific pricing, or 'org' for organization-specific pricing.
+    */
+    pricing_level String,
+
+    /*
+    `pricing_level_id`: The pricing level identifier for the token pricing. It can be 'default' or any other string(e.g. org id or project id). This allow with type to uniquely identify where the pricing applies.
+    */
+    pricing_level_id String,
+
+    /*
+    `provider`: The initial provider of the LLM, some LLMs are provided other than the default provider.
+    */
+    provider_id String,
```

Comment:
thank you for adding this - although currently it is unused, correct? Since we don't get provider out from our calls yet?

Reply:
yes, but we do have that info in the table courtesy of
https://github.com/AgentOps-AI/tokencost/blob/main/tokencost/model_prices.json

Reply:
nice! very good